app/src/main/java/org/thoughtcrime/securesms/home/search/GlobalSearchAdapterUtils.kt

```diff
@@ -99,8 +101,7 @@ fun ContentView.bindModel(query: String?, model: GroupConversation) {
     val groupRecipients = model.groupRecord.members.map { Recipient.from(binding.root.context, it, false) }
 
     val membersString = groupRecipients.joinToString {
-        val address = it.address.serialize()
-        it.name ?: "${address.take(4)}...${address.takeLast(4)}"
+        it.name ?: truncateIdForDisplay(it.address.serialize())
```

Comment:
Can this simply use the `getSearchName` defined below?

Reply:
nice, looks like.